File: C programming/Assignment/Assignment5/type4/pallindrome.c
#include <stdio.h>

int check(int num);
void main(){
    int num;
    printf("Enter the number:");
    scanf("%d",&num);
    if(check(num)==1)
        printf("pallindrome number");
    else    
        printf("not pallindrome number");

}
int check(int num){
    int sum,rev,mod;
  
    rev = num;
    while(num!=0){
        mod = num % 10;
        sum = sum * 10 + mod;
        num = num / 10;
        // printf("%d\n",sum);
    }

    if(sum == rev)
        return 1;
    return 0;
}

File: Java_programming/basic_java/assignment3/pallindrome.java
package assignment3;

public class pallindrome {

	public static void main(String[] args) {
		int num=121,sum=0,rev,mod;
//	    printf("Enter the number:");
//	    scanf("%d",&num);

	    rev = num;
	    while(num!=0){
	        mod = num % 10;
	        sum = sum * 10 + mod;
	        num = num / 10;
	        System.out.printf("%d\n",sum);
	    }

	    if(sum == rev){
	    	System.out.printf("The number is pallindrome number");
	    }
	    else{
	    	System.out.printf("The number is not pallindrome number");
	    }

	}

}
